### PR TITLE
Add new company - "Fly.io"

### PR DESCRIPTION
**Pull request by kellydosocorro**
This Pull Request should adhere to the repository's [Code of Conduct](../../main/.github/CONTRIBUTING.md).

Title your pull request descriptively. `[edit X company]` or `[add new company - COMPANY]` 

Please read and check the boxes in the list below that apply!

If updating or adding a company, please include details here: _delete this and add details_

Check off this list as appropriate (add an `x` between the `[]`). It will be used by @dougaitken when reviewing your contribution.

- [] This PR contains housekeeping only (URL edits, copy changes etc) - **if this is checked, delete other lines that don't apply.**
- [] You are an employee of the company mentioned and confirm all included details are correct.
- [X] You know your alphabet - company is listed in alphabetical order in the README.md file.
- [X] The company directly hires employees. ** No bootcamps / freelance sites / etc**
- [X] The company hires remote employees, or positions are available to remote workers and are clearly illustrated as such.
- [X] A [company profile](/company-profiles/example.md) is included - This is __required__ for new additions.
- [X] An entry has been added or edited within the [readme.md file](../../main/README.md).
- [X] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [X] __Region__ details the geographic regions in which this company employees can reside. For more details see the instructions in the [example company profile](/company-profiles/example.md#region).
- [X] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

This is a modified version of the [Contributing Guidelines](../../main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by KieranRobson — COMMENTED** [review]
The logs are failing as the .md file needs to be called `Fly.io` to match the title in the readme

**Comment by KieranRobson** [comment]
`5:54:56 PM: Fly.io.md: Company title "Fly.io" doesn't match filename (expected ~ "fly-io.md")` hmm the logs still fail

**Comment by kellydosocorro** [comment]
> The logs are failing as the .md file needs to be called `Fly.io` to match the title in the readme

Thanks for the suggestion, in this case, if I understand correctly, the file should be called literally "Fly.io.md" ?

File: company-profiles/fly-io.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+[Fly.io](https://fly.io/) is a platform for running full stack apps and databases close to your users (edge computing) all around the world.
+
+## Company size
+
+33 (as of October 2022)
+
+## Remote status
+
+Everyone is remote, but there is also an office in Chicago to who wants to use.
```
Comment (KieranRobson) [inline company-profiles/fly-io.md:13]: A readability improvement: 

`Everyone is remote, but there is an office in Chicago for those who would like to use it.`

File: README.md [hunk 2]
```diff
@@ -240,6 +240,7 @@ Name | Website | Region
 [FlightAware](/company-profiles/flightaware.md) | https://flightaware.com | USA
 [Flip](/company-profiles/flip.md) | https://flip.id | Indonesia
 [Flowing](/company-profiles/flowing.md) | https://flowing.it | Italy
+[Fly.io](/company-profiles/fly-io.md) | https://fly.io | USA
```
Comment (KieranRobson) [inline README.md:243]: Due to https://fly.io/jobs/ and `Current openings are for remote positions anywhere in the world.`, this probably should be `Worldwide`? or is the anywhere tag 'Anywhere within the USA'?

**Comment by dougaitken** [comment]
Hey @kellydosocorro looks like the name/file name is fixed now, can you take a look at @KieranRobson's review and amend as needed, please?

File: README.md (hunk 2 above)
Comment (kellydosocorro) [reply README.md:243]: Yeah, I think you are right, I will change this, thank you =D

**Comment by KieranRobson** [comment]
Looks good to me!

**Review by KieranRobson — APPROVED** [review]

File: company-profiles/fly-io.md (hunk 1 above)
Comment (kellydosocorro) [reply company-profiles/fly-io.md]: Thanks! I don't speak English natively so thank you for your suggestion, it turned out much better as you proposed.

**Comment by dougaitken** [comment]
Hey there, thanks for contributing, it is appreciated @kellydosocorro

**Pull request merged**